test/micro/org/openjdk/bench/java/util/concurrent/Maps.java

```diff
@@ -106,6 +116,12 @@ public void testConcurrentHashMap() {
         position = pos;
     }
 
+    @Benchmark
+    public void testCopyConstructor() {
+        ConcurrentHashMap<Integer, Integer> clone = new ConcurrentHashMap<>(staticMap);
+        dummy = clone;
```

Comment:
Is this for preventing dead-code elimination? If so, just do:

```
  return new ConcurrentHashMap<>(staticMap);
```

Reply:
I tried this. I don't think its working. When I just return the map, we lose the improvements in performance, and the benchmark overall just runs much faster. I'm guessing the map got DCE'ed

Reply:
Scratch that. Was my mistake. Was running the wrong test.

I'm renaming `testCopyConstructor` to `testConcurrentHashMapCopyConstructor`, since the file name + test don't actually explicitly say its for ConcurrentHashMap.